build_manpages/manpage.py

```diff
@@ -189,12 +188,25 @@ def _format_subparsers(self, action_group, action, subcommand=None):
 
         lines.append(self._format_ag_subcommands(action._choices_actions,
                      subcommand or self._prog))
+
+        # gather (sub-)command aliases
+        command_aliases = {}
+        command_aliases_names = set()
+        for name, command in action._name_parser_map.items():
+            if command not in command_aliases:
+                command_aliases[command] = []
+            else:
+                command_aliases[command].append(name)
+                command_aliases_names.add(name)
```

Comment:
Nice work here :-) I wish we had a test-case for this.

Reply:
done (in a different place)